src/client/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,19 @@
+.avatar-images-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
is the container actually needed? :)

Reply:
Yes, i think the container is needed, you can see explanations below.